src/main/java/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -14,5 +18,9 @@ public DefaultOperator(String name) {
 	}
 
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		RobotMap.HumanInput.Operator.joystick.button1.onlyWhileHeld(new ElevatorSetSpeed(RobotMap.HumanInput.Operator.joystick));
+		RobotMap.HumanInput.Operator.joystick.button2.onlyWhileHeld(new OverrideEnable(RobotMap.Component.elevator));
```

Comment:
overrideEnable and Disable are InstantCommands so they should be called whenPressed and whenReleased as opposed to onlyWhileHeld